Makefile

```diff
@@ -1,38 +1,38 @@
 GOTOOLS = \
 	github.com/mitchellh/gox \
-	github.com/golang/dep/cmd/dep \
 	github.com/golangci/golangci-lint/cmd/golangci-lint \
 	github.com/gogo/protobuf/protoc-gen-gogo \
 	github.com/square/certstrap
 GOBIN?=${GOPATH}/bin
 PACKAGES=$(shell go list ./...)
 OUTPUT?=build/tendermint
+GO111MODULE=on
```

Comment:
```suggestion
export GO111MODULE = on
```

Reply:
done

Reply:
That is required to be propagated to all commands - `Make` != `bash`